hips/utils/wcs.py

```diff
@@ -124,6 +124,30 @@ def create(cls, skydir: SkyCoord, shape: tuple, coordsys: str = 'CEL',
 
         return cls(w, shape)
 
+    @classmethod
+    def create_simple(cls, skydir: SkyCoord, shape: tuple, fov_deg: float,
+                      coordsys: str = 'CEL', projection: str = 'AIT') -> 'WCSGeometry':
```

Comment:
For `coordsys`, I would suggest we change this method and `create` to support `{'icrs', 'galactic'}`.
Having yet another code in the end-user API for the RADEC frame ("CEL" at the moment) is confusing.